### PR TITLE
Issue/clean up media removal and cleanup code

### PR DESCRIPTION
**Pull request by SergioEstevao**
Remove the cleanup media code, because besides not working properly is no longer needed because any media that is not upload to the server is still usable trough the media library. 
I also improve the removal of files when a media object is removed in order to not show error messages if the files where already remove in a previous step.

To test: 
 - Create a post
 - Add media to it (images or video), cancel the uploads on the middle of the process
- Check if no  logging messages are show referring to unable to removal files.
- If need check the Media folder inside the documents folder of the app to see if the files associated are removed.

Needs review: @astralbodies

### CODE REVIEW

**Comment by astralbodies** [comment]
What this doesn't clean up (any longer) are media files that are orphaned due to a Core Data migration issue and the model gets wiped. I *think* that's the original idea behind the method was to clean up the folder in case of some weird situations happening. Another example could be the app going weird when running out of disk space and the db gets in a bad state.

**Comment by SergioEstevao** [comment]
That are good points @astralbodies , I will redo the code and update it to clean the proper folders.

**Comment by SergioEstevao** [comment]
@astralbodies ready for another look, I updated the removal code to look inside the correct folder and clean up any image that isn't linked to a media object.

File: WordPress/Classes/Services/MediaService.h [hunk 1]
```diff
@@ -126,7 +126,11 @@
 #pragma mark - Media cleanup
 
 /**
- *  @brief      Removes all unused media files from the tmp directorys.
+ *  @brief      Removes all unused media files from the media directorys
```
Comment (astralbodies) [inline WordPress/Classes/Services/MediaService.h]: Just a simple spelling mistake - `directories`

**Comment by astralbodies** [comment]
Looks good - just the silly spelling mistake and then :shipit:

**Pull request merged**